Source/WebKit/UIProcess/RemoteLayerTree/mac/RemoteLayerTreeEventDispatcher.mm

```diff
@@ -646,18 +650,24 @@ void displayLinkFired(PlatformDisplayID displayID, DisplayUpdate, bool wantsFull
 void RemoteLayerTreeEventDispatcher::updateTimelineRegistration(WebCore::ProcessIdentifier processIdentifier, const HashSet<Ref<WebCore::AcceleratedTimeline>>& timelineRepresentations, MonotonicTime now)
 {
     assertIsHeld(m_animationLock);
-    if (!m_timelineRegistry)
-        m_timelineRegistry = makeUnique<RemoteAnimationTimelineRegistry>();
-    m_timelineRegistry->update(processIdentifier, timelineRepresentations, now);
-    if (m_timelineRegistry->isEmpty())
-        m_timelineRegistry = nullptr;
+    if (auto scrollingTree = this->scrollingTree())
+        scrollingTree->updateTimelineRegistration(processIdentifier, timelineRepresentations);
+    if (!m_monotonicTimelineRegistry)
+        m_monotonicTimelineRegistry = makeUnique<RemoteMonotonicTimelineRegistry>();
+    m_monotonicTimelineRegistry->update(processIdentifier, timelineRepresentations, now);
+    if (m_monotonicTimelineRegistry->isEmpty())
+        m_monotonicTimelineRegistry = nullptr;
 }
 
 const RemoteAnimationTimeline* RemoteLayerTreeEventDispatcher::timeline(const TimelineID& timelineID)
 {
     assertIsHeld(m_animationLock);
-    if (m_timelineRegistry)
-        return m_timelineRegistry->get(timelineID);
+    if (m_monotonicTimelineRegistry) {
+        if (RefPtr timeline = m_monotonicTimelineRegistry->get(timelineID))
+            return timeline.unsafeGet();
+    }
```

Comment:
This is the newly-added code that is triggering the style bot error because we're introducing a new call to `RefPtr::unsafeGet()`. This seemed preferable to me over adding an exception for the use of an unprotected raw pointer, but I'm not sure that it is. Perhaps we should make this whole method and related one return `RefPtr<RemoteAnimationTimeline>`?

Cc @cdumez, @ddkilzer and @geoffreygaren who may have an opinion.